Makefile

```diff
@@ -103,6 +103,18 @@ install:
 	install -C ./lotus-storage-miner /usr/local/bin/lotus-storage-miner
 	install -C ./lotus-seal-worker /usr/local/bin/lotus-seal-worker
 
+install-services:
+	install -C -m 0644 ./scripts/lotus-daemon.service /etc/systemd/system/lotus-daemon.service
+	install -C -m 0644 ./scripts/lotus-miner.service /etc/systemd/system/lotus-miner.service
```

Comment:
I think a better place for this would be: `/usr/local/lib/systemd/system`.
It is named:
> System units installed by the administrator 

`/etc/systemd/system` is for:
>  System units created by the administrator

Reply:
Fixed in https://github.com/filecoin-project/lotus/pull/1855